<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>320</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>168</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>205</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>158</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>196</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>267</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>293</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>237</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>QQ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>            背景图 片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit1</zorder>
  <zorder>lineEdit2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
